litellm/main.py

```diff
@@ -602,6 +604,7 @@ def completion(
         "cache",
         "no-log",
         "base_model",
+        "text_completion",
```

Comment:
Why text_completion ? 

The health check does honor `mode: completion` already when set as below:

```yaml
model_list:
 - model_name: starcoder2-3b
    litellm_params:
      api_base: https://vllm.example.com/v1
      api_key: "os.environ/API_KEY"
      model: openai/starcoder2-3b
      stream_timeout: 5
    model_info:
      mode: completion
```

Reply:
I tested with `text_completion: true`:

```yaml
    litellm_params:
      model: openai/mistral-7b-instruct
      api_base: https://api.siemens.com/llm/v1
      api_key: "os.environ/API_KEY"
      stream_timeout: 5
      text_completion: true
```

still the same